zerver/tests/test_realm.py

```diff
@@ -901,7 +901,9 @@ def test_creating_realm_creates_system_groups(self) -> None:
             "@role:everyone",
             "@role:internet",
         ]
-        self.assertEqual(user_group_names.sort(), expected_system_group_names.sort())
+        user_group_names.sort()
+        expected_system_group_names.sort()
+        self.assertEqual(user_group_names, expected_system_group_names)
```

Comment:
A simpler fix would be [`assertCountEqual`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual), which doesn’t require sorting.